src/__tests__/lib/containers/download_list/DownloadListTable.test.tsx

```diff
@@ -10,28 +10,19 @@ import {
   FileHandleAssociateType,
   PaginatedResults,
 } from '../../../../lib/utils/synapseTypes/'
-import * as React from 'react'
-import ReactDOM from 'react-dom'
-import { act } from 'react-dom/test-utils'
+import React from 'react'
+import { mount } from 'enzyme'
 import {
   MOCK_CONTEXT_VALUE,
   SynapseTestContext,
 } from '../../../../mocks/MockSynapseContext'
+import { resolveAllPending } from '../../../../lib/testutils/EnzymeHelpers'
 
 describe('it performs all functionality ', () => {
```

Comment:
This was another test giving me trouble. I decided to replace ReactDOM test utils with enzyme's mount, which was a simple change and seemed to resolve the issues I was encountering